src/panel/plugin/plugin-manager.vapi

```diff
@@ -0,0 +1,48 @@
+using Budgie;
```

Comment:
Any reason we aren't using gnome.generate_gir to create the vapi? as per https://github.com/BuddiesOfBudgie/budgie-desktop/blob/main/src/plugin/raven/meson.build#L60

Reply:
When I tried to, it failed to compile.

Reply:
To clarify, when generating the GIR, it can't find the symbols from `src/config`.